docs/user/benchmarking.md

```diff
@@ -20,7 +20,7 @@ Use `@wrap_experiment` to define your experiment. `ctxt` is used for
 The second and third parameters of the function should be `env_id` and `seed`.
 You should give your function a good name because it is used as the label name
 when it comes to plotting. See more [examples](https://github.com/rlworkgroup/garage/tree/master/benchmarks/src/garage_benchmarks/experiments).
-You can see detailed documentations of defining experiments [here](experiments.rst).
+You can see detailed documentations of defining experiments [here](experiments).
```

Comment:
The link doesn't work. You should probably keep `experiments.rst`?

Reply:
The master branch uses `experiments.rst` and it doesn't work. In the doc preview of this PR, I test it and it works.

Reply:
I see, thanks for clarifying.